### DISCUSSION TITLE
Unclear instructions on @vercel/next legacy mode

### DISCUSSION THREAD

**Discussion by mvanaltvorst** [vercel/vercel#5996] [Help]
This is the warning I'm getting during deploy on Vercel:
`WARNING: your application is being deployed in @vercel/next's legacy mode. http://err.sh/vercel/vercel/now-next-legacy-mode`

The document this error refers to states the following:
```
Why This Warning Occurred

@now/next has two modes: legacy and serverless. You will always want to use the serverless mode. legacy is to provide backwards compatibility with previous @now/next versions.
```

That seems to make sense. However, how exactly do I convert my project to a Serverless mode? I have added "serverless" to my target:

```
const { i18n } = require('./next-i18next.config');

module.exports = {
  target: 'serverless',
  i18n,
}
```

I'm using Next 10, so that should be fine as well.
I have replaced the normal "build" with "now-build":
```
  "scripts": {
    "dev": "next dev",
    "start": "next start",
    "now-build": "next build"
  },
```
But still, I'm getting the same legacy warning. What am I doing wrong?

**Comment by mcsdevv** [comment]
Hi @mvanaltvorst, the best option here is to remove all configuration related to `builders` as this is no longer required.
